en/tutorials-and-examples/blog/part-two.rst

```diff
@@ -261,7 +262,8 @@ ArticlesController::
 
         public function index()
         {
-            $this->set('articles', $this->Articles->find('all'));
+            $articles = $this->set('articles', $this->Articles->find('all'));
```

Comment:
Anto, I'm new to contributing and not well versed with GitHub at all... 
don't mean to inconvenience you.. but how do I push the changes I've 
made? I can't find how to do that directly on GitHub

On 22/05/2015 13:48, Anto wrote:
>
> In en/tutorials-and-examples/blog/part-two.rst 
> <https://github.com/cakephp/docs/pull/2766#discussion_r30911312>:
>
> > @@ -261,7 +262,8 @@ ArticlesController::
> >
> >          public function index()
> >          {
> > -            $this->set('articles', $this->Articles->find('all'));
> > +            $articles = $this->set('articles', $this->Articles->find('all'));
>
> @miguelarcher <https://github.com/miguelarcher> you either didn't push 
> your commit; either still not corrected the code example.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/cakephp/docs/pull/2766/files#r30911312>.
>